inc/integrations/class-coral.php

```diff
@@ -162,13 +159,18 @@ public function process_validate_endpoint_request( \WP_REST_Request $request ) {
 		// Verify the user's credentials.
 		$verified_data = apply_filters( 'wp_irving_verify_coral_user', $user_obj );
 
-		// Bail early if the verified user doesn't exist.
-		if ( empty( $verified_data ) || ! $verified_data ) {
+		// Bail early if the verified user isn't confirmed by the verification filter.
+		if (
+			empty( $verified_data ) ||
+			! $verified_data ||
```

Comment:
`empty()` evaluates `false` as empty, so I think this line is unnecessary. See: https://www.php.net/manual/en/function.empty.php